README.md

```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
     }
   })
   console.log(dimensions);
-
   await browser.close();
 })();
 ```
 
-## Contributing
+#### Intercept requests and wait for navigation
+Playwright can intercept network requests from a page to [modify or stub](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-request) them. Playwright scripts can also wait for [page navigations](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitfornavigationoptions), [selectors](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options), [network responses](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforresponseurlorpredicate-options), and [arbitrary predicates](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforfunctionpagefunction-options-args). This snippet intercepts network requests from a Firefox page, interacts with the page and waits for a navigation.
 
-Check out our [contributing guide](https://github.com/microsoft/playwright/blob/master/CONTRIBUTING.md).
 
-## FAQ
+```js
+const { firefox } = require('playwright');
+
+(async () => {
+  const browser = await firefox.launch({ headless: false });
+  const context = await browser.newContext();
+  const page = await context.newPage('http://todomvc.com');
+
+  // Intercept all network requests and log their URL
+  page.route('**', request => {

```

Comment:
Users on 0.10 will paste it and it won't work. Let's publish this with 0.11

Reply:
updated snippet to work with 0.10 as we are doing for the other snippets